src/AppBundle/Action/DefaultAction.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Action;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class DefaultAction
+{
+    use ControllerTrait;
+
+    private $container;
```

Comment:
I've no opinion about that but it's not what we do in symfony/symfony and it's not compliant with our current CS. Let's discuss this in another issue.